### PR TITLE
FE_user_chat_cta_component

### PR DESCRIPTION
**Pull request by sebas1222**
image
on click on 3 dots shows user cta options:

### CODE REVIEW

File: frontend/src/app/test/page.tsx [hunk 1]
```diff
@@ -152,6 +153,11 @@ export default function TestPage() {
                     userhandle="XokasXD"
                 />
             </div>
+            <WUserCHATCTA
+                userName="XokasXD"
+                userHandle="XokasXD"
```
Comment (AnthonyRicse98) [inline frontend/src/app/test/page.tsx:158]: delete props , use defaultProps

File: frontend/src/components/molecules/UserChatCTA/index.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { Avatar } from '@mui/material'
+import './index.scss'
+
+interface WUserCHATCTAProps {
+    avatarURL: string
+    userName: string
```
Comment (AnthonyRicse98) [inline frontend/src/components/molecules/UserChatCTA/index.tsx:7]: use operator ? 
example avatarUrl? : string  
in all cases

File: frontend/src/components/molecules/UserChatCTA/index.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                        <li>Cerrar Sesión</li>
+                        <li>Ver Perfil</li>
+                    </ul>
+                )}
+            </div>
+        </div>
+    )
+}
+
+export default WUserChatCTA
```
Comment (AnthonyRicse98) [inline frontend/src/components/molecules/UserChatCTA/index.tsx:37]: useDefaultProps
example 
```js

WUserChatCTA.defaultProps {
avatarURL: 'htttp:/.......'
}

File: frontend/src/components/molecules/UserChatCTA/index.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import { Avatar } from '@mui/material'
+import './index.scss'
+
+interface WUserCHATCTAProps {
+    avatarURL: string
+    userName: string
+    userHandle: string
+}
+
+const WUserChatCTA = ({ avatarURL = '', userName = 'User', userHandle = 'user' }: WUserCHATCTAProps) => {
```
Comment (RicseDev) [inline frontend/src/components/molecules/UserChatCTA/index.tsx:11]: ({ avatarURL, userName , userHandle } : WUser..

**Pull request merged**